packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -86,6 +86,8 @@ const PostTypeToTag: Record<PostType, FunctionComponent> = {
   [PostType.Share]: SharePostCard,
   [PostType.Welcome]: WelcomePostCard,
   [PostType.Freeform]: WelcomePostCard,
+  // TODO: replace the tag below with the right post card
```

Comment:
Todo that needs to be replaced when https://github.com/dailydotdev/apps/pull/2360 is merged